google-cloud-core/src/test/java/com/google/cloud/PolicyV3Test.java

```diff
@@ -46,24 +46,22 @@
       ImmutableList.of(USER, SERVICE_ACCOUNT, ALL_USERS);
   private static final List<String> MEMBERS_LIST_2 =
       ImmutableList.of(ALL_AUTH_USERS, GROUP, DOMAIN);
+
+  private static final Binding VIEWER_BINDING =
+      Binding.newBuilder().setRole(VIEWER).setMembers(MEMBERS_LIST_1).build();
+  private static final Binding EDITOR_BINDING =
+      Binding.newBuilder().setRole(EDITOR).setMembers(MEMBERS_LIST_2).build();
   private static final List<Binding> BINDINGS_NO_CONDITIONS =
-      ImmutableList.of(
-          Binding.newBuilder().setRole(VIEWER).setMembers(MEMBERS_LIST_1).build(),
-          Binding.newBuilder().setRole(EDITOR).setMembers(MEMBERS_LIST_2).build());
+      ImmutableList.of(VIEWER_BINDING, EDITOR_BINDING);
+  private static final Condition CONDITION =
+      Condition.newBuilder()
+          .setTitle("Condition")
+          .setDescription("Condition")
+          .setExpression("Expr")
+          .build();
   private static final List<Binding> BINDINGS_WITH_CONDITIONS =
       ImmutableList.copyOf(BINDINGS_NO_CONDITIONS)
-          .of(
-              Binding.newBuilder()
-                  .setRole(VIEWER)
-                  .setMembers(MEMBERS_LIST_1)
-                  .setCondition(
-                      Condition.newBuilder()
-                          .setTitle("Condition")
-                          .setDescription("Condition")
-                          .setExpression("Expr")
-                          .build())
-                  .build(),
-              Binding.newBuilder().setRole(EDITOR).setMembers(MEMBERS_LIST_2).build());
+          .of(VIEWER_BINDING.toBuilder().setCondition(CONDITION).build(), EDITOR_BINDING);
```

Comment:
`ImmutableList.of()` is a static method. Here you're calling a static method on an instance (returned by `ImmutableList.copyOf()`), which basically throws the instance away.

I think you can delete line 63 and just do `ImmutableList.of(VIEWER_BINDING.toBuilder()…, EDITOR_BINDING)`.

Reply:
done